numpy/core/tests/test_multiarray.py

```diff
@@ -6517,6 +6517,7 @@ def test_max_dims(self):
         a = np.empty((1,) * 32)
         self._check_roundtrip(a)
 
+    #pytest.mark.skipif(sys.version_info < (2, 7, 7), reason="ctypes issue")
```

Comment:
Should be `@`. May as well link to the bpo bug in the reason.

Reply:
gaaah, changed